src/scss/components/_badges.scss

```diff
@@ -11,51 +11,101 @@
  * https://fozzie.just-eat.com/styleguide/ui-components/badges
  */
 
-$badge-bgColor      : $grey--offWhite !default;
-$badge-lightBgColor : $white !default;
-$badge-infoColor    : $green !default;
-$badge-warningColor : $orange !default;
-$badge-alertColor   : $red !default;
-$badge-padding      : 4px spacing(x2) !default;
-$badge-radius       : 14px !default;
+$badge-bgColor              : $grey--offWhite !default;
+$badge-light-bgColor        : $white !default;
+$badge-info-color           : $green !default;
+$badge-success-color        : $green !default;
+$badge-success-bgColor      : $green--offWhite !default;
+$badge-warning-color        : $orange !default;
+$badge-alert-color          : $red !default;
+$badge-important-color      : $white !default;
+$badge-important-bgColor    : $orange !default;
+$badge-award-color          : $purple !default;
+$badge-award-bgColor        : $purple--light !default;
+$badge-padding              : 1px 5px !default;
+$badge-rounded-padding      : 4px spacing(x2) !default;
+$badge-rounded-radius       : 14px !default;
 
 @mixin badge() {
 
     .c-badge {
         background-color: $badge-bgColor;
         padding: $badge-padding;
+        position: relative;
 
         span {
             margin: 0 4px;
         }
     }
 
         .c-badge--rounded {
-            border-radius: $badge-radius;
+            border-radius: $badge-rounded-radius;
+            padding: $badge-rounded-padding;
         }
 
         .c-badge--light {
-            background-color: $badge-lightBgColor;
+            background-color: $badge-light-bgColor;
 
             span {
                 color: $grey--light;
             }
         }
 
         .c-badge--info {
-            color: $badge-infoColor;
+            color: $badge-info-color;
+        }
+
+        .c-badge--success {
+            background-color: $badge-success-bgColor;
+            color: $badge-success-color;
         }
 
         .c-badge--warning {
-            color: $badge-warningColor;
+            color: $badge-warning-color;
         }
 
         .c-badge--alert {
-            color: $badge-alertColor;
+            color: $badge-alert-color;
+        }
+
+        .c-badge--important {
+            background-color: $badge-important-bgColor;
+            color: $badge-important-color;
+        }
+
+        .c-badge--award {
+            background-color: $badge-award-bgColor;
+            color: $badge-award-color;
         }
 
         .c-badge--large {
             @include font-size(base--scaleUp);
         }
 
+        .c-badge--angled {
+            display: inline-block;
+            overflow: hidden;
+            padding-left: 15px;
+            padding-right: 15px;
+
+            &:before {
+                content: '';
+                border: 25px solid $white;
```

Comment:
is there anything we can do here for the border colour to inherit from the parent? 
Just thinking if the parents background is black: 
<img width="21" alt="screen shot 2018-08-31 at 13 33 36" src="https://user-images.githubusercontent.com/5295718/44912583-9ed43200-ad22-11e8-8ef8-bb56fc489d7f.png">

Reply:
Yeah, that would be good, I couldn't figure out a way to do that though. Any ideas?

Reply:
hmm, maybe instead of using border we could skew the `before` and `after`, like so:

```
&:after {
                content: '';
                width: 15px;
                height: 100%;
                background: inherit;
                position: absolute;
                right: -5px;
                top: 0;
                transform: skewX(-20deg);
            }
```

creates:
<img width="59" alt="screen shot 2018-08-31 at 14 00 00" src="https://user-images.githubusercontent.com/5295718/44913764-2ec7ab00-ad26-11e8-856f-a5fed3f09a6e.png">

this _might_ work?

Reply:
OK, so this works but isn't quite ideal

```css
.c-badge--angled {
    display: inline-block;
    transform: skew(-15deg);

    span {
        display: block;
        transform: skew(15deg);
    }
}
```

```html
<p class="c-badge c-badge--large c-badge--angled c-badge--alert"><span>Alert</span></p>
```

Note the nested span.

Reply:
If you remove the internal padding and add a little margin to allow the skew, i get this:

<img width="75" alt="screen shot 2018-08-31 at 14 39 43" src="https://user-images.githubusercontent.com/5295718/44915783-cc71a900-ad2b-11e8-8035-06e6ab2d1b7a.png">

from:
`<div class="c-badge c-badge--angled c-badge--award">hey</div>`

(ignore the colours, i fudged them for a test)

```
.c-badge--angled {
            display: inline-block;
            margin: 0 10px;

            &:before,
            &:after {
                background-color: inherit;
                content: '';
                height: 100%;
                position: absolute;
                top: 0;
                transform-origin: 100% 0px;
                transform: skew(-15deg);
                width: 12px;
            }
```

Reply:
that does add more padding, within the badge, than we probably want thought 🤔 - feel free to ignore my last comment 😄

Reply:
That's it! 🏁 

I owe you many 🍻's! 

![screen shot 2018-08-31 at 14 52 04](https://user-images.githubusercontent.com/89059/44916403-8f0e1b00-ad2d-11e8-9c2b-93ba75b888a2.png)